index.html

```diff
@@ -5543,26 +5544,42 @@ <h2 id="AudioWorklet-Sequence">
             </figcaption>
           </figure>
           <p>
-            The steps depicted in the diagram are one possible sequence of events
-            involving the creation of an <a>AudioContext</a> and an
-            associated <a>AudioWorkletGlobalScope</a>, followed by the creation of an
-            <a>AudioWorkletNode</a> and its associated <a>AudioWorkletProcessor</a>.
+            The steps depicted in the diagram are one possible sequence of
+            events involving the creation of an <a>AudioContext</a> and an
+            associated <a>AudioWorkletGlobalScope</a>, followed by the creation
+            of an <a>AudioWorkletNode</a> and its associated
+            <a>AudioWorkletProcessor</a>.
           </p>
           <ol>
-            <li>In the main scope, <code>window.audioWorklet</code> is requested
-            to import a script. No <a>AudioWorkletGlobalScope</a>s exist yet, so the
-            script is fetched and added to the Worklet module responses map.</li>
-            <li>An <a>AudioContext</a> is created.</li>
-            <li>An <a>AudioWorkletGlobalScope</a> is created in association with the context's audio rendering thread.</li>
-            <li>As part of the global scope's initialization, the set of imported module scripts is run, including the one that was previously imported.</li>
-            <li>As part of running the imported script, an <a>AudioWorkletProcessor</a> is registered under the
-            key <code>"Custom1"</code> within the <a>AudioWorkletGlobalScope</a>.</li>
-            <li>In the main scope, an <a>AudioWorkletNode</a> is created using the key <code>"Custom1"</code> along
-            with an <code>opts</code> dictionary of options.</li>
-            <li>As part of the node's creation, this key is used to look up the correct <a>AudioWorkletProcessor</a>
-            subclass for instantiation.</li>
-            <li>An instance of the <a>AudioWorkletProcessor</a> subclass is instantiated with a structured clone of the same
-            <code>opts</code> dictionary.</li>
+            <li>In the main scope, <code>window.audioWorklet</code> is
```

Comment:
I am not entirely sure why this non-normative section is necessary. Being user-friendly is certainly nice, but this information is too non-normative and the algorithm above is good enough to explain what's happening technically. @rtoy @padenot WDYT?